### PR TITLE
add FW Scripts to extend DB

### PR DESCRIPTION
**Pull request by astlexander**

### CODE REVIEW

**Comment by astlexander** [comment]
I have add FlyWay scripts to create DB accordantly to DB-schema showed in daily. 
In new directory "documentation" you can find screenshot of ERD Schema of created from PGAdmin

**Review by AlecKotovichSAM — COMMENTED** [review]
rename to jpeg

File: src/main/resources/db/migration/V1.4__rebuild_db_with_rename_role_table.sql [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+CREATE TABLE employeeroles
+(
+    employee_roles_id integer NOT NULL,
+    employee_id integer NOT NULL,
+    roles_id integer NOT NULL,
+    CONSTRAINT employeeroles_pkey PRIMARY KEY (employee_roles_id)
+);
+
+DROP TABLE IF EXISTS logon CASCADE;
+CREATE TABLE logon
```
Comment (AlecKotovichSAM) [inline src/main/resources/db/migration/V1.4__rebuild_db_with_rename_role_table.sql:33]: Rename to login

File: src/main/resources/db/migration/V1.4__rebuild_db_with_rename_role_table.sql [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    ADD CONSTRAINT roles_id_fkey FOREIGN KEY (roles_id)
+        REFERENCES roles (roles_id) MATCH SIMPLE
+        ON UPDATE NO ACTION
+        ON DELETE NO ACTION
+        NOT VALID;
+CREATE INDEX IF NOT EXISTS fki_roles_id_fkey
+    ON employeeroles(roles_id);
+
+
+ALTER TABLE logon
```
Comment (AlecKotovichSAM) [inline src/main/resources/db/migration/V1.4__rebuild_db_with_rename_role_table.sql:82]: login

**Review by AlecKotovichSAM — CHANGES_REQUESTED** [review]
I made comments, please fix

**Pull request merged**